examples/introductory_examples/example_sectors.py

```diff
@@ -0,0 +1,219 @@
+# definition, we know that this grid point does not lie `inside` the sector but on its border.
+# To prevent this inclusion of the bordering points, we may change the sector's
+# :attr:`~acoular.grids.SingleSector.include_border` attribute to ``False``.
+
+sector.include_border = False
+mask = sector.contains(grid.pos)
+print('x: {}, y: {}'.format(*grid.pos[:2, mask]))
+
+# %%
+# Note that the same point is still contained by the sector. This is because of the
+# :attr:`~acoular.grids.SingleSector.default_nearest` attribute, which – if no grid
+# points lie inside the sector – picks the grid point closest to the sector's center.
+# By setting it to ``False`` we can use only points that definitely lie `inside` the
+# sector – which are none in this case.
+
+sector.default_nearest = False
+mask = sector.contains(grid.pos)
+print('x: {}, y: {}'.format(*grid.pos[:2, mask]))
+
+# %%
+# Also, the tolerance for deciding whether a grid point lies on a sector's border can be
+# changed by varying the sector's :attr:`~acoular.grids.SingleSector.abs_tol` attribute.
+#
+# By defining which grid points lie inside the sector this way, we determine which points
+# the beamformer's :meth:`~acoular.fbeamform.BeamformerBase.integrate` method and Acoular's
+# :func:`~acoular.fbeamform.integrate` function sum over.
+
+# %%
+# =====================
+# Integrate Over Sector
+# =====================
+#
+# Now for the main event: integrating the beamforming result over our sector. This step gives
+# us the total sound pressure level (SPL) coming from just our region of interest. We can use
+# the beamformer's :meth:`~acoular.fbeamform.BeamformerBase.integrate` method or Acoular's
+# :func:`~acoular.fbeamform.L_p` function to get the sector's SPL.
+#
+# Right now, the sector does not contain any grid points and therefore the SPL will be zero for
+# all frequencies. To reinsert the ``(-0.3, 0.0)`` point into the sector and get beamforming
+# results, we need to change either the :attr:`~acoular.grids.SingleSector.default_nearest`
+# attribute or the :attr:`~acoular.grids.SingleSector.include_border` attribute back to ``True``.
+#
+# We look at the spectrum of the SPL over the power spectrum's FFT frequencies.
+
+sector.default_nearest = True
+
+freqs = f.fftfreq()
+bf_sector = bb.integrate(sector)
+spl_sector = ac.L_p(bf_sector)
+spl_sector = np.where(spl_sector > 0, spl_sector, 0)  # Keep positive entries only
+
+plt.figure(figsize=(8, 5))
+
+plt.plot(freqs / 1000, spl_sector)
+
+plt.xlabel('Frequency / kHz')
+plt.ylabel('$L_p$ / dB')
+plt.title('Integrated spectrum over sector')
+plt.grid()
+plt.show()
+
+
+# %%
+# ========================
+# Third-Octave Barspectrum
+# ========================
+#
+# Using Acoular's :func:`~acoular.tools.helpers.barspectrum` function
+# we can easily convert our integrated spectrum into third-octave bands.
+
+f_borders, spl_bars, f_center = barspectrum(spl_sector, freqs, 3, bar=True)
```

Comment:
The issue with the different results is probably due to this line. I am pretty sure that `barspectrum` expects the spectrum and not SPL. If the level is computed beforehand, the averaging not going to be correct. We should calculate the SPL after `barspectrum`.